### ISSUE TITLE
flux-sched needs testsuite and CI support

### ISSUE THREAD

**Issue by grondo** [flux-framework/flux-sched#3]
The flux-sched framework project (and other framework projects) should get tests and travis-ci integration like flux-core. In fact, we should have some scheme to easily add this support to all framework projects.

**Comment by lipari** [comment]
I just committed for review a unit test suite for the sched/resrc API:
https://github.com/lipari/flux-sched/tree/resrc_replacement
The tests themselves are not new.  They were taken from resrc/test/trescr.c and re-written to use TAP.
The mods to the resrc/Makefile to integrate the TAP framework are not very elegant.  Perhaps it is time to revisit the proposal to add autotools support to the sched project and take advantage of the infrastructure automake provides.

**Comment by dongahn** [comment]
This is a long standing issue. 

@SteVwonder plans to issue a PR shortly which captures most of what we discussed in this issue. 

Once that is mergrd, I think all of what we required are addressed except for autotools support. We still have to keep populating tests etc but that can now be done as a matter of course (i.e., without having add much infrastructure support). I propose we close this issue once @SteVwonder's PR is merged and create a new issue only for autotools support.

We have come a long way!
